js-api-spec/function.test.ts

```diff
@@ -77,6 +82,129 @@ it('passes a default argument value', () => {
   expect(fn).toHaveBeenCalled();
 });
 
+describe('simplifies', () => {
+  it('operations', () => {
+    const fn = spy(
+      () =>
+        new CalculationOperation(
```

Comment:
We shouldn't allow raw calculation operations (or interpolations) to be returned from functions. They aren't values in and of themselves, only when included within a `SassCalculation`.